python-sdk/src/astro/airflow/datasets.py

```diff
@@ -7,3 +9,26 @@
     # Airflow < 2.4
     Dataset = object
     DATASET_SUPPORT = False
+
+
+def kwargs_with_datasets(
+    kwargs: dict | None = None,
+    input_datasets: list[Dataset] | Dataset | None = None,
+    output_datasets: list[Dataset] | Dataset | None = None,
+):
+    """
+    Extract inlets and outlets from kwargs if users have passed it. If not, set input datasets as inlets and
+    set output dataset as outlets
```

Comment:
Do we have a note somewhere in the documentation that we support Datasets feature with python sdk on Airflow 2.4 ?

Reply:
+1

Reply:
Yeah I have that planned for a separate PR as mentioned in https://github.com/astronomer/astro-sdk/issues/611#issuecomment-1241402135